matpower/matpower-model/src/main/java/com/powsybl/matpower/model/MatpowerReader.java

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.matpower.model;
+
+import com.univocity.parsers.common.processor.BeanListProcessor;
+import com.univocity.parsers.tsv.TsvParser;
+import com.univocity.parsers.tsv.TsvParserSettings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.StringTokenizer;
+
+/**
+ * @author Christian Biasuzzi <christian.biasuzzi@techrain.eu>
+ */
+public class MatpowerReader {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MatpowerReader.class);
+
+    enum MatpowerSection {
+        BUS,
+        BRANCH,
+        GENERATOR
+    }
+
+    public static final String MATPOWER_SUPPORTED_VERSION = "2";
+
+    private String processCaseName(String str) {
```

Comment:
To be check if some or all of these functions could be static

Reply:
done:  changed this class' methods to static